Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml

```diff
@@ -0,0 +1,228 @@
+                    android:layout_height="1dp"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:background="?android:attr/listDivider"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_privacy_card" />
+
+
+                <TextView
+                    android:id="@+id/contact_diary_onboarding_privacy_information"
+                    style="@style/subtitle"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:focusable="true"
+                    android:text="@string/contact_diary_onboarding_legal_information"
+                    app:layout_constraintEnd_toEndOf="@id/guideline_end"
+                    app:layout_constraintStart_toStartOf="@id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_first_divider"
+                    tools:text="@string/contact_diary_onboarding_legal_information" />
+
+                <View
+                    android:id="@+id/contact_diary_onboarding_second_divider"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="1dp"
+                    android:layout_marginTop="@dimen/spacing_small"
+                    android:layout_marginBottom="20dp"
+                    android:background="?android:attr/listDivider"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    bind:layout_constraintEnd_toStartOf="@+id/guideline_end"
+                    bind:layout_constraintStart_toStartOf="@+id/guideline_start"
+                    bind:layout_constraintTop_toBottomOf="@+id/contact_diary_onboarding_privacy_information" />
+
+                <androidx.constraintlayout.widget.Guideline
+                    android:id="@+id/guideline_start"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    app:layout_constraintGuide_begin="@dimen/guideline_start" />
+
+                <androidx.constraintlayout.widget.Guideline
+                    android:id="@+id/guideline_end"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    app:layout_constraintGuide_end="@dimen/guideline_end" />
+
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+        </ScrollView>
+
+
+        <Button
+            android:id="@+id/contact_diary_onboarding_next_button"
```

Comment:
The spacing for the button should also use dimensions instead of hardcoded values.